src/amber.cr

```diff
@@ -18,7 +18,13 @@ require "./amber/websockets/**"
 
 module Amber
   class Server
+    include Amber::DSL::Server
     include Amber::Configuration
+    alias WebSocketAdapter = WebSockets::Adapters::RedisAdapter.class | WebSockets::Adapters::MemoryAdapter.class
+    property pubsub_adapter : WebSocketAdapter = WebSockets::Adapters::MemoryAdapter
+    property settings = Settings.new
```

Comment:
I'm fine with this being defined here but as long as we use configuration.cr doesn't it make more sense to define this there? In WIP settings branch you can call Settings.fetch(env) to get the env you need. Was thinking it would be easy to have `configuration#settings` return that..

Reply:
Agreed, it won't cause any issues to leave this line in `configuration.cr` will it? Should be functionally the same.

Reply:
I'll move this back to the confirguration.cr. Now bare in mind that this is a  Server property that is used within the server instance.